server/src/main/java/org/apache/gravitino/server/web/rest/ModelOperations.java

```diff
@@ -408,7 +408,7 @@ public Response getModelVersionByAlias(
           """
                     ANY(OWNER, METALAKE, CATALOG) ||
                     SCHEMA_OWNER_WITH_USE_CATALOG ||
-                    ANY_USE_CATALOG && ANY_USE_SCHEMA && (MODEL::OWNER || ANY_USE_MODEL && ANY_CREATE_MODEL_VERSION)
+                    ANY_USE_CATALOG && ANY_USE_SCHEMA && (MODEL::OWNER || ANY_USE_MODEL && ANY_LINK_MODEL_VERSION)
```

Comment:
How do you make backward compatibility when the user already stores the old privileges in the store?

Reply:
The legacy privileges already stored will be converted to new privileges when getting the role. You can see the code POConverters.